Schema.js

```diff
@@ -75,6 +78,7 @@ function classNameIsValid(className) {
     className === '_Session' ||
     className === '_SCHEMA' || //TODO: remove this, as _SCHEMA is not a valid class name for storing Parse Objects.
     className === '_Role' ||
+    className === '_GlobalConfig' ||
```

Comment:
This shouldn't really go here, as this is the place for the names of classes that the client is allowed to create. `_SCHEMA` is only here for legacy reasons, it should be removed.